Add resource accessibility to respa admin

urshala opened the pull request:
Allow admins to add/remove resource accessibility to resources.

In addition to gif below allow admins to add free text to accessibility description.


![resource_accessibility](https://user-images.githubusercontent.com/19978017/159456961-8c197685-3f39-4448-9c1c-9713cf4fa273.gif)

urshala committed:
Add resource accessibility to Resource serializer

The frontend needs to display the resource accessibility. There-
fore add the accessibility_summaries to the resource serializer.

Refs TAM-89

tomijarvi commented:
Didn't notice that there is already a ResourceAccessibility model which behaves differently than the Equipment that we talked about with the client. This should work similarly to Equipment selection. I sent the question to the client.

tomijarvi reviewed (approved):
The client wants to test this solution first in QA before we make any other changes.

urshala committed:
Add resource accessibility to respa admin

* Allow admins to add/remove resource accessibility to resources
  in respa admin.
* Add a new model field (accessibility_description) to resource
  model, which is used to fill in the accessibility items if not
  present in ResourceAccessibility model.

Refs TAM-89
